cypress/support/a11y-support.ts

```diff
@@ -1,27 +1,27 @@
 import 'cypress-axe'
 
-beforeEach(() => {
-  cy.injectAxe()
-})
+// beforeEach(() => {
```

Comment:
Need to restore this before this PR is done.

Reply:
@elevatebart Just an FYI that we will need to turn this back on before this PR gets merged. I turned it off because it was just making noise for me while I was building everything out initially.

Eventually, I would love to be able to remove this entirely in favor of the a11y checks that are new built into the Cloud.